heat/core/statistics.py

```diff
@@ -431,13 +435,66 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None):
     norm = x.shape[1] - ddof
     # find normalization:
     if norm <= 0:
-        raise ValueError("ddof >= number of elements in m, {} {}".format(ddof, m.gnumel))
+        raise ValueError("ddof >= number of elements in m, {} {}".format(ddof, m.numel))
     x -= avg.expand_dims(1)
     c = linalg.dot(x, x.T)
     c /= norm
     return c
 
 
+def kurtosis(x, axis=None, unbiased=True, Fischer=True):
```

Comment:
Can we have parameter and return type annotations here (except ``DNDarray`` due to aliasing)

Reply:
return type is DNDarray. will add a note in the docstring that it needs to be added